xlsxwriter/worksheet.py

```diff
@@ -423,11 +423,11 @@ def _write(self, row, col, *args):
             return self._write_number(row, col, *args)
         if token_type is str:
             return self._write_token_as_string(token, row, col, *args)
-        if token_type is unicode:
-            try:
+        try:
```

Comment:
Is this performance related? If not it should be in a separate PR.

Reply:
yes.  using type() is faster than isinstance, python3 doesn't have a unicode type so I had to work around that.